test6.js

```diff
@@ -0,0 +1,24 @@
+
+
+
+var potato = "";
+
+if (potato === "potato") {
+  console.log("hey you");
+}
+
+
+
+
+var foo = '' ;
+
+if(foo  === "bar") {
```

Comment:
Codacy has a fix for the issue: [Replace `(foo·` with `·(foo`](https://release.k8s.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=5796)
```suggestion
if (foo === "bar") {
```